packages/material-ui/src/Grid/Grid.test.js

```diff
@@ -24,68 +30,89 @@ describe('<Grid />', () => {
 
   describe('prop: container', () => {
     it('should apply the container class', () => {
-      const wrapper = shallow(<Grid container />);
-      expect(wrapper.hasClass(classes.container)).to.equal(true);
+      const { container } = render(<Grid container />);
+      const node = container.querySelector(`.${classes.container}`);
+
+      expect(node).to.not.equal(null);
```

Comment:
```suggestion
      expect(container.firstChild).to.have.class(classes.container);
```

*x* all the other assertions